h2o-py/h2o/h2o.py

```diff
@@ -451,16 +451,16 @@ def import_sql_table(connection_url, table, username, password, columns=None, op
     assert_is_type(password, str)
     assert_is_type(columns, [str], None)
     assert_is_type(optimize, bool)
-    assert_is_type(streaming, bool)
+    assert_is_type(fetch_mode, str, None)
     p = {"connection_url": connection_url, "table": table, "username": username, "password": password,
-         "optimize": optimize, "streaming": streaming}
+         "optimize": optimize, "fetch_mode": fetch_mode}
```

Comment:
please keep 'optimize' only in method signature above just to avoid errors from user's scripts.
it can safely be removed anywhere else from the code, especially from REST requests.